lib/src/main/java/com/diffplug/spotless/Architecture.java

```diff
@@ -0,0 +1,39 @@
+package com.diffplug.spotless;
+
+/**
+ * Enumeration of possible computer architectures.
+ *
+ */
+public enum Architecture {
```

Comment:
Make this package-private inside the `rome` package. I would also just remove the unsupported architectures, but I'm okay leaving them in if you want.

Reply:
Thanks very much for the review. I was thinking that the OS/architecture detection could be used by other formatters such as NPM, but it makes sense that you want to keep dependencies between formatters as low as possible. We can always refactor if or when other formatter also need to detect the OS/architecture. And yes, if it's only for Rome for now, unsupported architectures can be removed.